lib/ansible/modules/network/avi/avi_analyticsprofile.py

```diff
@@ -30,130 +30,154 @@
             - Greater than this number and the client's request is considered frustrated.
             - Allowed values are 1-30000.
             - Default value when not specified in API or module is interpreted by Avi Controller as 500.
+        type: int
     apdex_response_tolerated_factor:
         description:
             - Client tolerated response latency factor.
             - Client must receive a response within this factor times the satisfactory threshold (apdex_response_threshold) to be considered tolerated.
             - Allowed values are 1-1000.
             - Default value when not specified in API or module is interpreted by Avi Controller as 4.0.
+        type: float
     apdex_rtt_threshold:
         description:
             - Satisfactory client to avi round trip time(rtt).
             - Allowed values are 1-2000.
             - Default value when not specified in API or module is interpreted by Avi Controller as 250.
+        type: int
     apdex_rtt_tolerated_factor:
         description:
             - Tolerated client to avi round trip time(rtt) factor.
             - It is a multiple of apdex_rtt_tolerated_factor.
             - Allowed values are 1-1000.
             - Default value when not specified in API or module is interpreted by Avi Controller as 4.0.
+        type: float
     apdex_rum_threshold:
         description:
             - If a client is able to load a page in less than the satisfactory latency threshold, the pageload is considered satisfied.
-            - It is considered tolerated if it is greater than satisfied but less than the tolerated latency multiplied by satisfied latency.
+            - It is considered tolerated if it is greater than satisfied but less than the tolerated latency multiplied by satisifed latency.
             - Greater than this number and the client's request is considered frustrated.
-            - A pageload includes the time for dns lookup, download of all http objects, and page render time.
+            - A PageLoad includes the time for DNS lookup, download of all HTTP objects, and page render time.
             - Allowed values are 1-30000.
             - Default value when not specified in API or module is interpreted by Avi Controller as 5000.
+        type: int
     apdex_rum_tolerated_factor:
         description:
             - Virtual service threshold factor for tolerated page load time (plt) as multiple of apdex_rum_threshold.
             - Allowed values are 1-1000.
             - Default value when not specified in API or module is interpreted by Avi Controller as 4.0.
+        type: float
     apdex_server_response_threshold:
         description:
-            - A server http response is considered satisfied if latency is less than the satisfactory latency threshold.
+            - A server HTTP response is considered Satisfied if latency is less than the Satisfactory Latency Threshold.
```

Comment:
All the capitals seem odd, since elsewhere you're not capitalizing "satisfied". I'd stick with lower case for the most part. Also, the basic description of what the value means is missing. Maybe something like this?
```suggestion
            - Threshold for a server HTTP response.
            - A server HTTP response is considered satisfied if latency is less than the satisfactory latency threshold.
```